### PR TITLE
Add latiss_cwfs_align script.

### PR DESCRIPTION
**Pull request by tribeiro**
The idea is to move this script to standard scripts, but I still need to figure out how to unit test it. So, in the meantime, I'll keep it in external scripts.

### CODE REVIEW

File: python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py [hunk 1]
```diff
@@ -0,0 +1,547 @@
...
+from lsst import cwfs
+from lsst.cwfs.instrument import Instrument
+from lsst.cwfs.algorithm import Algorithm
+from lsst.cwfs.image import Image, readFile, aperture2image, showProjection
+import lsst.cwfs.plots as plots
+
+
+class LatissCWFSAlign(salobj.BaseScript):
+    """ Perform an optical alignment procedure of Auxiliary Telescope with
+    the GenericCamera CSC.
```
Comment (patrickingraham) [inline python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py]: Not with GenericCamera :)

File: python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py [hunk 2]
```diff
@@ -0,0 +1,547 @@
...
+    -----
+    **Checkpoints**
+
+    * intra: when taking intra focus image.
+    * extra: when taking extra focus image.
+    * cwfs: when running CWFS code.
+
+    **Details**
+
+    This is what the script does:
```
Comment (patrickingraham) [inline python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py]: Just add a 2 sentence description for now.

File: python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py [hunk 3]
```diff
@@ -0,0 +1,547 @@
...
+        if remotes:
+            self.attcs = ATTCS(self.domain)
+            self.latiss = LATISS(self.domain)
+
+        self.short_timeout = 5.
+        self.long_timeout = 30.
+
+        # Sensitivity matrix: mm of hexapod motion for nm of wfs. To figure out
+        # the hexapod correction multiply
+        self.sensitivity_matrix = [[-1. / 131., 0., 0.],
```
Comment (patrickingraham) [inline python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py]: Once we get this established we have to move these into configs in the AOS. Fine for now

File: python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py [hunk 4]
```diff
@@ -0,0 +1,547 @@
...
+
+    @property
+    def side(self):
+        return int(self._side * self.dz / 1.5)
+
+    @dz.setter
+    def dz(self, value):
+        self._dz = float(value)
+        cwfs_config_template = """#Auxiliary Telescope parameters:
+Obscuration 				0.3525
```
Comment (patrickingraham) [inline python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py]: shouldn't these be indented?

File: python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py [hunk 5]
```diff
@@ -0,0 +1,547 @@
...
+                                                     filter=self.filter, grating=self.grating)
+
+        azel = await self.attcs.atmcs.tel_mount_AzEl_Encoders.aget()
+        nasmyth = await self.attcs.atmcs.tel_mount_Nasmyth_Encoders.aget()
+
+        self.angle = np.mean(azel.elevationCalculatedAngle) + np.mean(nasmyth.nasmyth2CalculatedAngle)
+
+        self.log.debug("Move hexapod to zero position")
+
+        await self.hexapod_offset(-self.dz)
```
Comment (patrickingraham) [inline python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py]: add a comment to say this puts the hexapod back to it's original (in-focus) position

File: python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py [hunk 6]
```diff
@@ -0,0 +1,547 @@
...
+
+        self.log.info(f"extraImage expId for target: {self.extra_visit_id}")
+
+    async def hexapod_offset(self, offset):
+        """
+
+        Parameters
+        ----------
+        offset
+        use_ataos
```
Comment (patrickingraham) [inline python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py]: i think the use_ataos isn't there...

**Comment by tribeiro** [comment]
Merging this after getting confirmation with @patrickingraham ... He'll implement the fixes he suggested above along with adding a new feature to bin the data.

**Pull request merged**